src/model/input.ts

```diff
@@ -186,35 +186,35 @@ class Input {
   }
 
   static get sshPublicKeysDirectoryPath(): string {
-    return Input.getInput('sshPublicKeysDirectoryPath') || '';
+    return Input.getInput('sshPublicKeysDirectoryPath') ?? '';
```

Comment:
This hides errors and is a bit of an antipattern.

I really believe we shouldn't be converting all or operators to nullish coalesce.

Reply:
Just curious, doesn't the || do the same thing for hiding issues? I thought nullish was just a more robust version of ||.